Sources/_TestingInternals/include/Includes.h

```diff
@@ -58,7 +58,7 @@
 #include <sys/fcntl.h>
 #endif
 
-#if __has_include(<sys/resource.h>)
+#if __has_include(<sys/resource.h>) && !defined(__wasi__)
```

Comment:
What's the issue here? (I assume this fix is needed to unbreak Wasm.)

Reply:
Oh, I see it's in the commit message. n/m

Reply:
Including `sys/resource.h` from wasi-libc requires a feature macro to be defined (`-D_WASI_EMULATED_PROCESS_CLOCKS`) and emits compilation errors otherwise. However, the header is not used in WASI paths, so it is safe to exclude it from the list of includes.

See `sys/times.h` that is transitively included by `sys/resource.h` https://github.com/WebAssembly/wasi-libc/blob/wasi-sdk-22/libc-top-half/musl/include/sys/times.h